src/Illuminate/Validation/Rules/StringRule.php

```diff
@@ -252,6 +252,16 @@ public function same(string $field): static
         return $this->addRule('same:'.$field);
     }
 
+    /**
+     * The field under validation must be a valid A or AAAA record.
+     *
+     * @return $this
+     */
+    public function activeUrl(): static
+    {
+        return $this->addRule('active_url');
+    }
```

Comment:
Should this really be part of the string validation and not of a separate URI validation rule like the email validation rule?

Reply:
@mrvipchien @xoesae see #54519